doc/openshift-guide.md

```diff
@@ -184,22 +233,90 @@ You should now see a section **Production: Self-managed Gateway** at the bottom
+ 
+1. For each API, go to the **Integration** tab, and in _Production_ section enter the _Public Base URL_ and click on **Update Production Configuration** to save the changes. Make sure you use a different _Public Base URL_ for each API, for example, `http://search-api.openshift.demo` for the service called Search API and `http://video-api.openshift.demo` for Video API.
+
+2. In OpenShift create routes for the gateway service ("threescalegw"): `http://search-api.openshift.demo` and `http://video-api.openshift.demo`. From **Applications > Routes** you can create a new route or modify and existing one. Note that you can't change the hostname for already created routes, but you can delete an existing route and add a new one.
+
+ <img src="https://support-preview.3scale.net/images/screenshots/guides-openshift-create-more-routes.png" alt="Create routes for multiple services" >
+
+3. You will need to redeploy the gateway to apply the changes you've made in the 3scale admin portal. Go to **Applications > Deployments > threescalegw** and click on **Deploy**.
+
+4. Now you can make calls to both APIs, and they will be routed to either Search API or Video API depending on the hostname. For example:
 
-Your API is now protected by two instances of the 3scale API Gateway running on Red Hat OpenShift, following all the configuration that you set-up in the 3scale Admin Portal.
+ ```
+ curl "http://search-api.openshift.demo/find?query=openshift&user_key=YOUR_USER_KEY"
+ curl "http://video-api.openshift.demo/categories?user_key=YOUR_USER_KEY"
+ ```
+
+ In case you are running the OpenShift cluster running locally, in order for the above calls to work properly you will need to add the hostnames for your `OPENSHIFT-SERVER-IP` to the _/etc/hosts_ file:
+ ```
+ 172.30.0.112 search-api.openshift.demo
+ 172.30.0.112 video-api.openshift.demo
+ ```
+
+ In case your OpenShift cluster is hosted remotely, you can specify the host configured in _Public Base URL_ in the `Host` header in order to get it routed corectly by OpenShift and the API gateway:
+
+ ```
+ curl "http://YOUR-PUBLIC-IP/find?query=openshift&user_key=YOUR_USER_KEY" -H "Host: search-api.openshift.demo"
+ curl "http://YOUR-PUBLIC-IP/categories?user_key=YOUR_USER_KEY" -H "Host: video-api.openshift.demo"
+ ```
+
+### Changing APIcast parameters
+
+APIcast v2 gateway has a number of parameters that can enable/disable different features or change the behavior. This parameters are defined in the OpenShift template, you can find the complete list in the template YAML file. The template parameters are mapped to environment variables that will be set for each running pod.
+
+You can specify the values for the parameters when creating a new application with `oc new-app` command using the `-p | -- param` argument, for example:
+
+<pre><code>oc new-app -f https://github.com/3scale/apicast/releases/download/v2.0.0-rc1/openshift-template.yml -p APICAST_LOG_LEVEL=debug</code></pre>
```

Comment:
Imo we should point to the latest template, not to rc1.  Otherwise this gets out of date really quickly.